python/meterpreter/ext_server_stdapi.py

```diff
@@ -1000,6 +1000,13 @@ def stdapi_fs_file_move(request, response):
 	return ERROR_SUCCESS, response
 
 @meterpreter.register_function
+def stdapi_fs_file_copy(request, response):
+	oldname = packet_get_tlv(request, TLV_TYPE_FILE_NAME)['value']
+	newname = packet_get_tlv(request, TLV_TYPE_FILE_PATH)['value']
+	shutil.copy(unicode(oldname), unicode(newname))
```

Comment:
According to the python docs, shutil.copy also copies permission bits. is that what we want? Only asking because it's a slight semantic difference between the other implementations.